src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs

```diff
@@ -223,6 +267,11 @@ public bool IsWarningAsError
         /// </summary>
         internal virtual IReadOnlyList<string> CustomTags { get { return (IReadOnlyList<string>)this.Descriptor.CustomTags; } }
 
+        /// <summary>
+        /// Gets property bag for the diagnostic.
```

Comment:
Will do.


-          Heejae


From: Srivatsn Narayanan [mailto:notifications@github.com]
Sent: Monday, February 16, 2015 6:32 PM
To: dotnet/roslyn
Cc: HeeJae Chang
Subject: Re: [roslyn] add property bag to diagnostic and remember origin of the diagnostic (#510)


In src/Compilers/Core/Portable/Diagnostic/Diagnostic.cs<https://github.com/dotnet/roslyn/pull/510#discussion_r24788937>:

> @@ -223,6 +267,11 @@ public bool IsWarningAsError

>          /// </summary>

>          internal virtual IReadOnlyList<string> CustomTags { get { return (IReadOnlyList<string>)this.Descriptor.CustomTags; } }

>

> +        /// <summary>

> +        /// Gets property bag for the diagnostic.

Can you add a better doc comment? Maybe similar to the one in the Create method

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/510/files#r24788937>.